README.md

```diff
@@ -46,7 +46,18 @@ SDK support with custom configuration for kafka stream, that is :
 * DialTimeout : Timeout duration during connecting to kafka broker (time.Duration)
 * ReadTimeout : Timeout duration during consume topic from kafka broker (time.Duration)
 * WriteTimeout : Timeout duration during publish event to kafka broker (time.Duration) 
+* LogMode : eventstream will print log based on following levels: Info, Warn, Debug, Error and Off (event stream level) 
```

Comment:
does this and all above have default value?

Reply:
no, if user doesn't set logMode, we dont set the logrus level

Reply:
all of those have defaults, you know. and that defaults need to be communicated

Reply:
you're right, has been updated in latest commit
thanks